lib/tournament_cases.rb

```diff
@@ -0,0 +1,27 @@
+class TournamentCase < OpenStruct
+  def test_name
+    'test_%s' % description.tr(' ', '_').tr('()', '')
+  end
+
+  def skipped
+    index.zero? ? '# skip' : 'skip'
+  end
+
+  def work_load
```

Comment:
Great job adhering to the generator interface!  Through discussions over the last couple days it was decided to settle on using `workload` without the underscore.